charon/src/bin/charon-driver/main.rs

```diff
@@ -122,6 +122,18 @@ fn main() {
     compiler_args.push("-Copt-level=3".to_string());
     compiler_args.push("-Coverflow-checks=false".to_string());
     compiler_args.push("-Cdebug-assertions=false".to_string());
+    // Add some options to handle the attributes specific to Charon and Aeneas
+    // Note that the clients can also add the following attributes to their crates:
+    // ```
+    // #![feature(register_tool)]
+    // #![register_tool(charon)]
+    // ```
+    compiler_args.push("-Z".to_string());
+    compiler_args.push("crate-attr=feature(register_tool)".to_string());
+    compiler_args.push("-Z".to_string());
+    compiler_args.push("crate-attr=register_tool(charon)".to_string());
+    compiler_args.push("-Z".to_string());
+    compiler_args.push("crate-attr=register_tool(aeneas)".to_string());
```

Comment:
Can you remove the `register_rool` clauses from the top of the new test file as well?

Reply:
Ah, we can't register the same tool twice. I'm afraid this means we have to let our users set this themselves

Reply:
Yes.